x-pack/plugins/security_solution/public/common/components/event_details/insights/insights.tsx

```diff
@@ -73,7 +71,6 @@ export const Insights = React.memo<Props>(
       hasProcessSessionInfo;
 
     const canShowAncestryInsight =
-      isRelatedAlertsByProcessAncestryEnabled &&
       processEntityField &&
       processEntityField.values &&
```

Comment:
That one was necessary because TypeScript needed extra proof that the values exist, haha. I added a type guard in https://github.com/elastic/kibana/pull/138419/commits/24131f7be52465b2863577638e32d14dfa21443b so this code is not necessary anymore :)